app/redux/ui/actions.ts

```diff
@@ -1,5 +1,6 @@
-import { AppThDispatch } from '../../types';
+import { AppThDispatch, GetState } from '../../types';
 import { eventsService } from '../../infra/eventsService';
+import StoreService from '../../../desktop/storeService';
```

Comment:
Do not use things from the main process in the renderer process, and vice versa. Or move it to `shared` directory.

BUT, I'm not sure that now the Main process and Renderer process are the same and synced. You have to ensure that we have only one source of truth (one instance).